docs/Principios_Tests.md

```diff
@@ -0,0 +1,21 @@
+# Principios F.I.R.S.T

```

Comment:
Esto no aporta mucho. Si queréis documentar cómo se han seguido los principios, hacedlo como se documenta, en los mensajes de commit del código que se cree.

Reply:
Perdón pero no llego a entender la forma correcta de plasmar esto entonces. ¿Si al llevar a cabo los test he seguido estos 5 principios, en cada commit relacionado a un test debo explicar en ese mismo mensaje de commit como se han seguido dichos principios?

Reply:
O en los issues a partir de los que se cree, claro.

Reply:
En todo caso, "la forma correcta" es hacerlo y no contarlo. Ya veré yo si lo estáis haciendo o no.